kadas/gui/search/kadasalternategotolocatorfilter.cpp

```diff
@@ -0,0 +1,671 @@
+/***************************************************************************
+                        KadasAlternateGotoLocatorFilters.cpp
+                        ----------------------------
+   begin                : February 2025
+   copyright            : (C) 2025 by Denis Rouzaud
+   email                : denis@opengis.ch
+***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include <qgis/qgsannotationlayer.h>
+#include <qgis/qgsfeedback.h>
+#include <qgis/qgsmapcanvas.h>
+#include <qgis/qgscoordinateutils.h>
+#include <qgis/qgscoordinatereferencesystemutils.h>
+#include <qgis/qgsannotationmarkeritem.h>
+#include <qgis/qgsmarkersymbollayer.h>
+#include <qgis/qgsmarkersymbol.h>
+
+#include "kadasalternategotolocatorfilter.h"
+#include "kadas/core/kadaslatlontoutm.h"
+
+
+KadasAlternateGotoLocatorFilter::KadasAlternateGotoLocatorFilter( QgsMapCanvas *mapCanvas, QObject *parent )
+  : QgsLocatorFilter( parent )
+  , mCanvas( mapCanvas )
+{
+  QString degChar = QString( "%1" ).arg( QChar( 0x00B0 ) );
```

Comment:
```suggestion
  QString degChar = QStringLiteral( "%1" ).arg( QChar( 0x00B0 ) );
```
Should be done in pretty much the whole file